rln-wasm/README.md

```diff
@@ -0,0 +1,20 @@
+# RLN for WASM
+This library is used in [waku-org/js-rln](https://github.com/waku-org/js-rln/)
+
+## Building the library
+1. Make sure you have nodejs installed and the `build-essential` package if using ubuntu.
+2. Install wasm-pack
+```
+curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+```
+3. Compile zerokit for `wasm32-unknown-unknown`:
+```
+cd rln-wasm
+wasm-pack build --release
+```
```

Comment:
Maybe add a pointer to how https://github.com/waku-org/js-rln/? Maybe this belongs more in that repo, but looking at README not obvious to me how to run something similar to the example app that existed here before